satpy/readers/satpy_cf_nc.py

```diff
@@ -282,6 +282,27 @@ def _coordinate_datasets(self, configured_datasets=None):
             self.fix_modifier_attr(ds_info)
             yield True, ds_info
 
+    def _compare_attr(self, _ds_id_dict, key, data):
+        if key in ['name', 'modifiers']:
+            return True
+        elif key == 'wavelength':
+            if _ds_id_dict[key] != WavelengthRange.from_cf(data.attrs[key]):
+                return False
+        else:
+            if data.attrs[key] != _ds_id_dict[key]:
+                return False
+        return True
+
+    def _attrs_equal(self, ds_id, data):
```

Comment:
```suggestion
    def _dataid_attrs_equal(self, ds_id, data):
```

Reply:
I don't we have a helper like this unfortunately.